src/main/java/uk/gov/hmcts/reform/sandl/snlnotes/Application.java

```diff
@@ -1,12 +1,38 @@
 package uk.gov.hmcts.reform.sandl.snlnotes;
 
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+import org.springframework.web.cors.CorsConfiguration;
+import org.springframework.web.cors.UrlBasedCorsConfigurationSource;
+import org.springframework.web.filter.CorsFilter;
 
 @SpringBootApplication
+@EnableJpaAuditing
 @SuppressWarnings("HideUtilityClassConstructor") // Spring needs a constructor, its not a utility class
 public class Application {
     public static void main(String[] args) {
         SpringApplication.run(Application.class, args);
     }
+
+    @Bean
+    public CorsFilter corsFilter(
+        @Value("${management.security.allowedOrigin}") String allowedOrigins,
+        @Value("${management.security.allowedHeaders}") String allowedHeaders
+    ) {
+        CorsConfiguration config = new CorsConfiguration();
+        config.setAllowCredentials(true);
+        config.addAllowedOrigin(allowedOrigins);
+        config.addAllowedHeader(allowedHeaders);
+        config.addAllowedMethod("GET");
+        config.addAllowedMethod("POST");
+        config.addAllowedMethod("PUT");
+
+        UrlBasedCorsConfigurationSource source = new UrlBasedCorsConfigurationSource();
+        source.registerCorsConfiguration("/logout", config);
```

Comment:
@leszekgu Do we need to registerCours for /logout if there is no such endpoint?

Reply:
we don't, it should be removed